app/announcement/models/Need.py

```diff
@@ -2,33 +2,45 @@
 Module defining the Need model.
 
 This module contains the Need class, which represents a specific need or requirement
-in the application, inheriting from the LabeledIcon base class.
+in the application base class.
 """
 
 from django.db import models
 
-from app.core.models import LabeledIcon
 from app.core.models.ArchitectSpeciality import ArchitectSpeciality
 
 
-class Need(LabeledIcon):
+class Need(models.Model):
```

Comment:
why are you removing the LabeledIcon class?

Reply:
In inserting data for models inheriting from LabeledIcon it becomes unnecessarily complicated, mist add instance for LabeledIcon and then create child instance and point to it , and when all fixtures depend from LabeledIcon there can be conflicts if ptr values overlapse